src/nucleic_acid/nucleic_acid.jl

```diff
@@ -41,6 +41,23 @@ Q_{T, A} & Q_{T, C} & Q_{T, G} & Q_{T, T} \\end{bmatrix}\$
                    β * πT, α * πT, γ * πT, -(β * πA + α * πC + γ * πG))
 end
 
+function Q(mod::NASM, scaled::Bool)
```

Comment:
Pedantic, but can we use `scale` as the argument name. With `scaled` I find if it's ambigious whether it is an indication if something has already been scaled vs. an instruction for the function to perform the scaling. To me it is a bit more clear with an argument `scale` that it is instructive.

Reply:
I have no problem changing the name to `scale`, but it shadows the name of the function `scale`, leading to an error when `P_generic` calls `scale` on line 72:

    ERROR: MethodError: objects of type Bool are not callable

I can change the call on line 72 to `scale_generic(mod)` or `SubstitutionModels.scale(mod)` to prevent this.

Reply:
Right, of course. We could use `_scale()` as the function, and not export as another option. It seems to me that calculating the scale outside of the P and Q functions now would not be common.